src/stages/main/patchers/SwitchCasePatcher.js

```diff
@@ -139,13 +139,14 @@ export default class SwitchCasePatcher extends NodePatcher {
   /**
    * @private
    */
-  getBreakToken(): ?SourceToken {
-    let lastToken = this.sourceTokenAtIndex(this.contentEndTokenIndex);
-    if (lastToken && lastToken.type === BREAK) {
-      return lastToken;
-    } else {
-      return null;
+  hasExistingBreak(): boolean {
+    if (!(this.consequent instanceof BlockPatcher)) {
+      return false;
     }
+    let lastStatement = this.consequent.statements[this.consequent.statements.length - 1];
+    return lastStatement &&
+        lastStatement.node.type === 'Identifier' &&
+        lastStatement.node.data === 'break';
```

Comment:
Related, it seems wrong that this is an identifier and not a `BreakStatement` or something.